web/viewer.css

```diff
@@ -1509,7 +1509,8 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
   display: none;
 }
 
-@media screen and (min-resolution: 2dppx) {
+@media screen and (min-resolution: 2dppx),
+       screen and (-webkit-min-device-pixel-ratio: 2) {
```

Comment:
> I don't think your suggestion is valid CSS (and it's not accepted by browsers).

I didn't test, or look up anything; it was just a quick and untested suggestion. (Also, I don't have Safari to test with.)

> Is it confirmed not-working at your end?

Please note that the *main* use case for the default viewer, is to be the UI for the version of PDF.js that is included in Firefox. Hence this is the one thing that a PR is really not allowed break.

Below is a screenshot of what happens in a `node make {firefox|mozcentral}` build with this patch applied, when the browser is switched to HiDPI mode. Note how the *entire* viewer completely breaks.

![no_hidpi](https://cloud.githubusercontent.com/assets/2692120/7831024/e2566c2c-0452-11e5-864d-d215c7ad2cb0.png)